tidb-cloud/cppo-customer.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Reseller's Customer
+summary: Learn how to become a reseller's customer.
+---
+
+# Reseller's Customer
+
+A reseller's customer subscribes to TiDB Cloud services through a reseller in the AWS Marketplace.
+
+Unlike direct TiDB Cloud customers, a reseller's customers pay invoices via the AWS Marketplace rather than directly to PingCAP. However, all other daily operations in the TiDB Cloud console remain the same for both AWS CPPO and direct TiDB Cloud customers.
+
+This document outlines how to check both future and historical bills.
+
+## Migrate from a direct TiDB Cloud account to a reseller's customer account
+
+> **Tip:**
+>
+> Direct customers are the end customers who purchase TiDB Cloud and pay invoices directly from PingCAP.
+
+If you are currently a direct customer with a TiDB Cloud account, you can ask your reseller to migrate your account to a reseller's customer account.
+
+The migration will take effect on the first of a future month. Discuss with your reseller to determine the specific effective date.
+
+On the effective day of migration, you will receive an email notification.
+
+## View your future bills
+
+As a reseller's customer, your invoices will be processed and paid through the AWS Marketplace.
```

Comment:
Made some edits because the subject of "As a ..." is "you".
```suggestion
As a reseller's customer, you will have your invoices processed and paid through the AWS Marketplace.
```